### DISCUSSION TITLE
2007/05/16/%E8%B0%83%E6%95%B4%E6%8E%A7%E4%BB%B6%E5%92%8C%E5%87%BD%E6%95%B0%E9%9D%A2%E6%9D%BF%E7%9A%84%E9%A6%96%E9%80%89%E9%A1%B9

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#159] [Announcements]
# 2007/05/16/%E8%B0%83%E6%95%B4%E6%8E%A7%E4%BB%B6%E5%92%8C%E5%87%BD%E6%95%B0%E9%9D%A2%E6%9D%BF%E7%9A%84%E9%A6%96%E9%80%89%E9%A1%B9

LabVIEW 在 8.0 版对控件和函数面板作了一次较大调整。LabVIEW 功能越来越强大，控件和函数面板上的东西越来越多。如果增加面板的嵌套深度，用户每次选取面板上的一个控件和函数都要多点几下鼠标，而且对不熟悉位置的东西找起来也相当费劲；如果扩充每一个面板上的容量，一个面板上图标太多，用户会眼花，也不利于查找。所以 LabVIEW 8 调整了面板的显示方式：最顶层面板所有栏目都以文字的方式显示，纵向排成一列。其中第一个栏目是默认就展开的，可以直接看到次级面板的内容；其它栏目都收起，鼠标挪上去，才看得到它里面的内容。

https://blog.qizhen.xyz/2007/05/16/%E8%B0%83%E6%95%B4%E6%8E%A7%E4%BB%B6%E5%92%8C%E5%87%BD%E6%95%B0%E9%9D%A2%E6%9D%BF%E7%9A%84%E9%A6%96%E9%80%89%E9%A1%B9

**Comment by ruanqizhen** [comment]
之前博客下的留言：

Zhang Ping
2007年07月25日 09:24 编辑
LV升级太快了，用新版本编出来的程序旧版本都打不开， LabVIEW 8.5  我晕啊

回复
俊
2007年05月18日 11:03 编辑
@_@
 
 非我等专业，就是看的吃力啊

回复
vivi笑了
2007年05月17日 21:13 编辑
好专业哦~~~强悍ni ^0^

回复
秋
2007年05月17日 09:43 编辑
梦境~
米看懂~
 

回复
Qizhen
2007年05月17日 06:38 编辑
我 blog 上的内容主要来自于和其他人的讨论以及自己的实践总结。如果是被普遍介绍的知识，我也就没必要再费劲得重写一遍了。另外，本篇技巧由 Austin 的李楠mm传授，在此表示感谢:)

回复
杰
2007年05月16日 22:10 编辑
我学LV主要看2本书，一本是铁道出版社的，LabVIEW7 express 使用技术教程，还有一本是清华出版社的LabVIEW高级程序设计。完全没有这方面的内容，以及你的blog里面的很多技术内幕。
其他的学习主要靠实践和NI的资料，以及Developer Zone。
如果不看你的blog，象你所说的这样的小技巧，能通过什么方式获得？

回复
